vidgrid-recorder-cli.sln

```diff
@@ -1,14 +1,16 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.1062
+# Visual Studio Version 16
```

Comment:
Is this visual studio 2017 or 2019? I have the build instructions for 2017 not sure if this will break anything

Reply:
I believe it's 2019. I started the CLI originally on 2019. I don't know how 2017 left its mark here but it shouldn't matter either way. The build for the CLI is pretty straightforward since you just build it and it outputs it to the same directory as the recorder